okhttp/src/main/java/okhttp3/internal/http2/Http2Connection.kt

```diff
@@ -489,9 +489,10 @@ class Http2Connection internal constructor(builder: Builder) : Closeable {
    *
    * @param sendConnectionPreface true to send connection preface frames. This should always be true
    *     except for in tests that don't check for a connection preface.
+   * @param daemon whether to use a Daemon thread for the connection.
    */
   @Throws(IOException::class) @JvmOverloads
-  fun start(sendConnectionPreface: Boolean = true) {
+  fun start(sendConnectionPreface: Boolean = true, daemon: Boolean = true) {
```

Comment:
FWIW I don't see this as an anti-pattern in Kotlin, so would like to leave them.  But in java tests it should be made clearer.